fennel/client/client.py

```diff
@@ -301,6 +301,22 @@ def extract_historical_features(
         else:
             return pd.Series(response.json())
 
+    def lookup(
+        self, dataset_name: str, keys: List[Dict[str, Any]], fields: List[str]
+    ) -> tuple[Union[pd.DataFrame, pd.Series], pd.Series]:
+        req = {
+            "keys": keys,
```

Comment:
Now that we have this function, can we modify tests in client_tests/test_dataset.py to replace Dataset.lookup to client.lookup ?

Reply:
Took a brief look, I don't think it's that straightforward. The `client.lookup` requires the running of the our def server.